tools/build/rules/mm.bzl

```diff
@@ -25,11 +25,18 @@ def mm_library(name, srcs=[], hdrs = [], copy_hdrs=[], **kwargs):
     for src in srcs:
         copy_name = name+"_copy_"+src
         copied_srcs += [":"+copy_name]
-        copy(
-            name = copy_name,
-            src = src,
-            dst = name+"/"+src+".cc",
-        )
+        if src.endswith(".mm"):
+            copy(
+                name = copy_name,
+                src = src,
+                dst = name+"/"+src+".cc",
```

Comment:
This is copying `foo.mm` to `foo.mm.cc`?
Do you understand what this rule is used for? If so, can we put some comments in here?

Reply:
Yes, that is exactly what it does, because bazel doesn't like compiling .mm files (it has very strict rules on file extensions). Objective-C/C++ stuff is compiled with the default c++ compiler, but passing the "-x objective-c++" option to clang.

I will add some comments to this.